src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/ManifestReader.cs

```diff
@@ -86,7 +86,9 @@ private static void ReadMetadataValue(ManifestMetadata manifestMetadata, XElemen
                         manifestMetadata.Id = value;
                         break;
                     case "version":
-                        manifestMetadata.Version = NuGetVersion.Parse(value);
+                        NuGetVersion version;
```

Comment:
How does pack behave when someone passes an actual empty or unparseable version on the commandline?

Reply:
We do a `NuGetVersion.TryParse` and if that fails we throw:

```Error NU5010: Version string specified for package reference ' ' is invalid.```
```Error NU5010: Version string specified for package reference 'sdf' is invalid.```

Empty string:
```Missing option value for: '-Version'```

Reply:
Can you link where that happens?

Reply:
Here: https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.CommandLine/Commands/PackCommand.cs#L131-L137